h/migrations/versions/9389d52b037d_add_index_to_annotation_thread_root.py

```diff
@@ -0,0 +1,27 @@
+"""
+add_index_to_annotation_thread_root
```

Comment:
It's a tiny thing, but this is usually a plain language sentence describing the change, such as "Add index to annotation thread root". At the moment this looks a bit funny:

```
$ hypothesis --dev migrate history | head -n10
b102c50b1133 -> 9389d52b037d (head), add_index_to_annotation_thread_root
50df3e6782aa -> b102c50b1133, Clean up moderation extra key
e554d862135f -> 50df3e6782aa, add annotation moderation table
5655d56d7c29 -> e554d862135f, Add index to flag.user_id
c322c57b49db -> 5655d56d7c29, Add flag table
18dfed902c9e -> c322c57b49db, Remove user uid column
7e2443f8d7d6 -> 18dfed902c9e, Remove unused user indices
faefe3b614db -> 7e2443f8d7d6, Make userid index unique
1a40e75a524d -> faefe3b614db, Make user uid nullable
02db2fa6ea98 -> 1a40e75a524d, Add normalised username index
```